eng/Version.Details.xml

```diff
@@ -9,51 +9,61 @@
 -->
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.AspNetCore.Razor.Language" Version="3.0.0-preview4.19123.2">
+    <Dependency Name="Microsoft.AspNetCore.Razor.Language" Version="3.0.0-preview4.19124.1">
+      <CoherentParentDependency>Microsoft.NETCore.App</CoherentParentDependency>
```

Comment:
As currently implemented, these should be attributes on Dependency, not child elements.

Reply:
@mmitche :ok: Guess I thought this file was more like a MSBuild `.props` file.